rcongui/src/components/ChatWidget/ChatWidget.js

```diff
@@ -1,35 +1,26 @@
-import React, { useEffect, useRef, useState } from "react";
+import React, {useEffect, useRef, useState} from "react";
 import Badge from "@material-ui/core/Badge";
-import { Comment, Send } from "@material-ui/icons";
-import {
-  Chip,
-  Drawer,
-  Grid,
-  makeStyles,
-  TextField,
-  Button,
-  Fab
-} from "@material-ui/core";
+import {Comment, Send} from "@material-ui/icons";
+import {Box, Button, Chip, Drawer, Grid, makeStyles, TextField} from "@material-ui/core";
 import IconButton from "@material-ui/core/IconButton";
 import Typography from "@material-ui/core/Typography";
 import moment from "moment";
 
 const useStyles = makeStyles((theme) => ({
   message: {
     whiteSpace: "pre-wrap",
-    textAlign: "left"
+    textAlign: "left",
+    marginTop: '5px',
+    marginBottom: '5px',
```

Comment:
Please us material UI's spacing. The whole framework is about getting the spacing consistant :)